scripts/dev/gen_stub.php

```diff
@@ -84,6 +90,31 @@ public static function fromNode(Node $node) {
         throw new Exception("Unexpected node type");
     }
 
+    public function getFastZpp($argName) {
+        if ($this->isBuiltin) {
+            switch (strtolower($this->name)) {
+                case "bool":
+                    return sprintf("Z_PARAM_BOOL%s(%s)", $this->isNullable ? "_OR_NULL" : "", $argName);
+                case "int":
+                    return sprintf("Z_PARAM_LONG%s(%s)", $this->isNullable ? "_OR_NULL" : "", $argName);
+                case "float":
+                    return sprintf("Z_PARAM_DOUBLE%s(%s)", $this->isNullable ? "_OR_NULL" : "", $argName);
+                case "string":
+                    return sprintf("Z_PARAM_STRING%s(%s, %s)", $this->isNullable ? "_OR_NULL" : "", $argName, $argName . "_length");
+                case "array":
+                    return sprintf("Z_PARAM_ARRAY%s(%s)", $this->isNullable ? "_OR_NULL" : "", $argName);
+                case "object":
+                    return sprintf("Z_PARAM_OBJECT%s(%s)", $this->isNullable ? "_OR_NULL" : "", $argName);
+                case "callable":
+                    return sprintf("Z_PARAM_FUNCT%s(%s)", $this->isNullable ? "_OR_NULL" : "", $argName);
+                default:
+                    throw new Exception("Not implemented: $this->name");
+            }
+        } else {
```

Comment:
Fell free to invert the condition `$this->isBuiltin` to `$this->isBuiltin === false` and remove the `else` here 💅